package.json

```diff
@@ -30,7 +30,7 @@
     "lerna": "^3.22.1",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.8.3",
-    "typescript": "~4.7.4"
+    "typescript": "~5.1.1-rc"
```

Comment:
5.1 just went public :)

Reply:
yeah, I've got it in my local branch. It has a problem with master I need to solve.

Also, typescript-eslint still warns about >=5.1.0 so I figure we wait for that to be fixed before landing. I'll drop this to Draft.